src/docs/BUILD.md

```diff
@@ -3,85 +3,200 @@
+    macOS:
+    ```sh
+    export PYTHONPATH=<GENAI_ROOT_DIR>/build:$PYTHONPATH
+    export DYLD_LIBRARY_PATH=<GENAI_ROOT_DIR>/build/openvino_genai:$DYLD_LIBRARY_PATH
+    ```
 
-1. Build and install OpenVINO from sources following the [instructions](https://github.com/openvinotoolkit/openvino/wiki#how-to-build)  
-The path to the openvino install directory is referred as <INSTALL_DIR> throughout the document.
-2. Clone OpenVINO GenAI repository and init submodules:
+    Windows Command Prompt:
+    ```cmd
+    set PYTHONPATH=<GENAI_ROOT_DIR>\build;%PYTHONPATH%
+    set PATH=<GENAI_ROOT_DIR>\build\openvino_genai;%PATH%
+    ```
+
+    Windows PowerShell:
+    ```sh
+    $env:PYTHONPATH = "<GENAI_ROOT_DIR>\build;$env:PYTHONPATH"
+    $env:PATH = "<GENAI_ROOT_DIR>\build\openvino_genai;$env:PATH"
+    ```
+
+To optimize the package size, you can reduce the ICU (International Components for Unicode) data size when building OpenVINO Tokenizers.
+For more information please refer to the [OpenVINO Tokenizers instructions](https://github.com/openvinotoolkit/openvino_tokenizers?tab=readme-ov-file#reducing-the-icu-data-size).
+
+
+### Build OpenVINO GenAI Wheel
+
+1. Clone OpenVINO GenAI repository and init submodules:
     ```sh
     git clone --recursive https://github.com/openvinotoolkit/openvino.genai.git
     cd openvino.genai
     ```
-3. Build the project:
+2. Set up the environment:
+    - Option 1 - using OpenVINO `setupvars.sh` script:
+        ```sh
+        source <INSTALL_DIR>/setupvars.sh
+        ```
+    - Option 2 - setting environment variables manually:
+        ```sh
+        export OpenVINO_DIR=<INSTALL_DIR>/runtime
+        export PYTHONPATH=<INSTALL_DIR>/python:./build/:$PYTHONPATH
+        export LD_LIBRARY_PATH=<INSTALL_DIR>/runtime/lib/intel64:$LD_LIBRARY_PATH
+        ```
+3. Upgrade pip to ensure you have the latest version:
     ```sh
-    source <INSTALL_DIR>/setupvars.sh
-    cmake -DCMAKE_BUILD_TYPE=Release -S ./ -B ./build/
-    cmake --build ./build/ --config Release --target package -j
-    cmake --install ./build/ --config Release --prefix ov
+    python -m pip install --upgrade pip
+    ```
+4. Build the wheel in the `dist` directory:
+    ```sh
+    python -m pip wheel . -w dist/ --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/pre-release
+    ```
+
+### Install OpenVINO GenAI From Source
+
+1. Clone OpenVINO GenAI repository and init submodules:
+    ```sh
+    git clone --recursive https://github.com/openvinotoolkit/openvino.genai.git
+    cd openvino.genai
+    ```
+2. Set up the environment:
+    - Option 1 - using OpenVINO `setupvars.sh` script:
+        ```sh
+        source <INSTALL_DIR>/setupvars.sh
+        ```
+    - Option 2 - setting environment variables manually:
+        ```sh
+        export OpenVINO_DIR=<INSTALL_DIR>/runtime
+        export PYTHONPATH=<INSTALL_DIR>/python:./build/:$PYTHONPATH
+        export LD_LIBRARY_PATH=<INSTALL_DIR>/runtime/lib/intel64:$LD_LIBRARY_PATH
```

Comment:
Please add the same split for other OS.
I just remembered, OPENVINO_LIB_PATHS is required for windows by core ov for python. Please add it to every env description including `Build OpenVINO, OpenVINO Tokenizers, and OpenVINO GenAI From Source` path.

Reply:
Added `OPENVINO_LIB_PATHS` for windows.
Commands for other OS discussed - will be kept for now as is for building wheels and python installation